retrofit-converters/logansquare/src/test/java/retrofit/converter/MyObject.java

```diff
@@ -0,0 +1,46 @@
+package retrofit.converter;
+
+import com.bluelinelabs.logansquare.annotation.JsonField;
+import com.bluelinelabs.logansquare.annotation.JsonObject;
+
+@JsonObject()
+public class MyObject {
+    @JsonField
+    public String message;
```

Comment:
Yeah, the fields and getters can have default scope.  Do I need to set getters to default scope?